data/src/emulator.js

```diff
@@ -929,13 +929,13 @@ class EmulatorJS {
                 }
             },
             totalDependencies: 0,
-            monitorRunDependencies: () => {},
-            locateFile: function(fileName) {
+            monitorRunDependencies: () => { },
```

Comment:
Here's another example of a space between a bracket where it doesnt really make much sense (now that I think about it, why is `monitorRunDependencies` even defined? Does this do anything?)